settings.gradle

```diff
@@ -52,12 +52,6 @@ include 'functional_test:weave_test'
 include 'functional_test:weave_test_impl'
 
 // JDK 9+ module system tests
-if (JavaVersion.current().isJava9Compatible()) {
```

Comment:
These only caused failures locally, right?
I think we should leave these for GHA for now.

Reply:
um...not failures on GHA yet WITH these changes